{"uuid": "7898260e-8452-471b-b963-a5cb3781fdd9", "children": ["3c07f5d4-e083-44e1-986d-2a4671385334"], "befores": [{"name": "classObjects", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"C:\\Users\\Yerkebulan\\Desktop\\projects\\Lab\\AppiumLearn\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Yerkebulan\\Desktop\\projects\\Lab\\AppiumLearn\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Yerkebulan\\Desktop\\projects\\Lab\\AppiumLearn\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Yerkebulan\\Desktop\\projects\\Lab\\AppiumLearn\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Yerkebulan\\Desktop\\projects\\Lab\\AppiumLearn\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 884, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Yerkebulan\\Desktop\\projects\\Lab\\AppiumLearn\\AppiumFrameWork\\tests\\LoginTest.py\", line 12, in classObjects\n    self.bp = BasePage(self.driver)\n              ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1717654972599, "stop": 1717654972599}], "start": 1717654972599, "stop": 1717654972605}